.github/workflows/release-layer-ruby.yml

```diff
@@ -19,12 +19,20 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Build
-        id: save-ruby-sdk-version
         run: |
           ./build.sh
         shell: bash
         working-directory: ruby/src
 
+      - name: Save Ruby SDK Version
+        id: save-ruby-sdk-version
+        run: |
+          unzip -q build/opentelemetry-ruby-layer.zip
+          export GEM_PATH=$PWD/ruby/gems/3.2.0/
+          RUBY_SDK_VERSION=$(ruby -e 'require "opentelemetry-sdk"; puts OpenTelemetry::SDK::VERSION')
+          echo "RUBY_SDK_VERSION=$RUBY_SDK_VERSION" >> $GITHUB_OUTPUT
+        working-directory: ruby/src
```

Comment:
Mind testing it on a personal fork?  You can temporarily disable the call to the publish workflow.

Reply:
Sure, although I already did similar test on my [test repo](https://github.com/xuan-cao-swi/test-action-release/actions/runs/10009575266), but I am happy to do it again on my personal fork.

Do I just comment out the `publish_layer` job and add `workflow_dispatch:` for testing, and revert back once it's ready?

Reply:
That's good enough for me.